src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/GetRecallNotificationComponentTest.kt

```diff
@@ -114,7 +166,9 @@ class GetRecallNotificationComponentTest : ComponentTestBase() {
         vulnerabilityDiversity = true,
         vulnerabilityDiversityDetail = "Some stuff",
         assessedByUserId = userId,
-        inCustody = false
+        inCustody = inCustody,
+        arrestIssues = !inCustody,
```

Comment:
Checking: this is temporary, pending actual property?

Reply:
I hadn't planned to. it was just an easy workaround the changing inCustody field 🤷‍♂️